src/config/Config.h

```diff
@@ -8,6 +8,10 @@
 #include "window/Window.h"
 
 namespace LUS {
+class ConfigVersionUpdater {
+  public:
+    virtual void Update(Config* conf) = 0;
```

Comment:
I think it's a more flexible system if this class stores a version. That's how it works in the other places we store versioning.

Reply:
are you thinking it would store the version that the config is being updated *from* or the version it's being updated *to*?

Reply:
The version we're updating from. That makes for the easiest way to tell that we are on the latest version. If there's no version updater class matching the version we are on, then we are on the latest version. Or there's a missing version updater which should definitely be considered an error.